react/features/base/tracks/actionTypes.js

```diff
@@ -10,15 +10,45 @@
 export const TRACK_ADDED = Symbol('TRACK_ADDED');
 
 /**
- * Action for when a track cannot be created because permissions have not been
- * granted.
+ * Action triggered when a local track starts being created through the WebRTC
+ * getUserMedia call. It will include extra 'gumProcess' field which is
+ * a Promise with extra 'cancel' method which can be used to cancel the process.
+ * Canceling will result in disposing any JitsiLocalTrack returned by the GUM
+ * callback. There will be TRACK_CREATE_CANCELED event instead of track
+ * added/gum failed events.
  *
- * {
- *     type: TRACK_PERMISSION_ERROR,
- *     trackType: string
+ * @type {
```

Comment:
Why is `@type` used here? I don't think we've used it before in the jsdocs of action types because here we're describing the payload of the action and not the type of the `const`'s value. I don't mind us adopting its use (though I'd appreciate hearing more about why it makes sense here) but I don't support using it only for the action types of this PR because that introduces inconsistency.

Reply:
There was no specific reason, I think I've copy pasted description of action from another place